src/features/Instructors/InstructorDetailTable/columns.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/prop-types, no-nested-ternary */
+import React from 'react';
+import { format } from 'date-fns';
+import { Link } from 'react-router-dom';
+import { Badge } from 'react-paragon-topaz';
+
+const columns = [
+  {
+    Header: 'Class',
+    accessor: 'className',
+    Cell: ({ row }) => (
+      <Link
+        to={`/courses/${row.values.masterCourseName}/${row.values.className}?classId=${row.original.classId}`}
+        className="link"
+      >
+        {row.values.className}
+      </Link>
+    ),
+  },
+  {
+    Header: 'Course',
+    accessor: 'masterCourseName',
+  },
+  {
+    Header: 'Start - End Date',
+    accessor: 'startDate',
+    Cell: ({ row }) => {
+      const startDate = row.values.startDate ? format(row.values.startDate, 'MM/dd/yy') : '';
+      const endDate = row.original.endDate ? format(row.original.endDate, 'MM/dd/yy') : '';
+      return <div>{startDate} - {endDate}</div>;
+    },
+  },
+  {
+    Header: 'Status',
+    accessor: 'status',
+    Cell: ({ row }) => (
+      row.values.status === 'pending'
```

Comment:
Please see columns in ClassPage to define status in a variable. When is more than 2 nested if, is difficult to understand what is the condition